frontend/src/slices/auth/auth.slice.ts

```diff
@@ -12,8 +12,13 @@ type State = {
 };
 
 const initialState: State = {
+  user: {
+    id: 1,
+    email: 'test@gmail.com',
+    firstName: 'John',
+    lastName: 'Doe',
+  },
```

Comment:
Mock it up when you need it - not in store
```suggestion
  user: null,
```

Reply:
So should I replace mocked data to Profile component?

Reply:
Yep, I guess it'd be better to mock it in Profile component